tests/www/test_views.py

```diff
@@ -755,6 +755,7 @@ def setUp(self):
         from airflow.www.views import dagbag
         from airflow.utils.state import State
         dag = DAG(self.DAG_ID, start_date=self.DEFAULT_DATE)
+        dag.sync_to_db()
```

Comment:
Why do we add many sync_to_db method to different tests?  I saw this kinda change in many of your prs.

Reply:
Good point @feng-tao. Apparently, without the syncing, the tests become flakey. I think we should solve the root cause of this, but this might not be trivial.

Reply:
I've done some work on making each test be stand-alone. And without it when running individual tests many tests would fail if the Scheduler tests haven't run, so this sort of thing makes each test less dependent on the other tests, or the order the test are run in.